docpages/01_frequently_asked_questions.md

```diff
@@ -3,87 +3,113 @@
 
 ## Does this library support Visual Studio 2022?
-Yes! The master branch comes with pre-built binaries for Visual Studio 2022 and our Windows bot template has a [vs2022 branch](https://github.com/brainboxdotcc/windows-bot-template/tree/vs2022) which you can clone to get Visual Studio 2022 specific code. For the time being we support both Visual Studio 2019 and 2022. At some point in the future only 2022 may be supported as 2019 becomes outdated.
+
+Yes! The master branch comes with pre-built binaries for Visual Studio 2022 and our Windows bot template has a [VS2022 branch](https://github.com/brainboxdotcc/windows-bot-template/tree/vs2022) which you can clone to get Visual Studio 2022 specific code. For the time being we support both Visual Studio 2019 and 2022. At some point in the future only 2022 may be supported as 2019 becomes outdated.
 
 ## How much of the library is completed?
+
 All REST calls (outbound commands) are completed including all currently available interactions, and all Discord events are available. The library also has voice support.
 
 ## How do I chat with the developers or get help?
+
 The best place to do this is on the [Discord server](https://discord.gg/dpp). You most likely won't get an answer immediately (we have lives, and need to sleep sometimes), but we will be willing to help!
 
 ## How can I contribute to development?
-Just star and fork a copy of the repository, and submit a Pull Request! We won't bite! Authors of accepted pull requests get a special role on our [Discord server](https://discord.gg/dpp).
 
-## Whats the best way to learn C++?
+Just star and fork a copy of the repository, and submit a Pull Request! We won't bite! Significant contributors get a special role on our [Discord server](https://discord.gg/dpp).
+
+## What's the best way to learn C++?
+
 A simple search can find some learning tools, however not all are good. Here is a list of some (good) learning resources:
 
-* [CodeAcademy](https://www.codecademy.com/learn/c-plus-plus)
+* [Codecademy](https://www.codecademy.com/learn/c-plus-plus)
 * [Learn CPP](https://www.learncpp.com/)
 * [Learn CPP (Very Basic)](https://www.learn-cpp.org/)
 
-If you don't understand something then feel free to ask in the [Discord server](https://discord.gg/dpp) ...*we don't bite!*
+If you don't understand something then feel free to ask in the [Discord server](https://discord.gg/dpp)… *we don't bite!*
```

Comment:
im not sure we should be using this three-dots unicode? I've never seen it used anywhere in docs, and i dont even have any ability to type it on my keyboard.

Reply:
I didn't even realise that was three dots, i can't see it without my glasses on. I think we should stick to the three dots.

Reply:
Well, ellipsis is the correct typography. An easy to spot eyesore for me. I'm aware it's not present on standard keyboard layouts, text processors typically insert it automatically if you hit dot three times, text editors typically need a bit of help. I insert it using a Unicode plugin which I also use for the likes of ©, various arrows, and well, bunch of other Unicode (often ASCII really, just not present on a typical keyboard layout) characters. It's one of these things I'm always happy to fix if I run into it, and that's not end of the world if it gets missed somewhere. Ofc if it's not welcome, I can roll these back. It's minimal difference for end user either way.

Reply:
yeah not sure its good, as its hard to read for people with poor eyesight.

Reply:
> yeah not sure its good, as its hard to read for people with poor eyesight. to me it almost looks like "_" (i wear glasses)

That's what I thought it was without my glasses lol

Reply:
It actually renders as three dots that are slightly more spaced than if you just type ... in the actual presentation, but yeah, they get kinda slapped together in monospace due to well, space reasons.